showcase.json

```diff
@@ -3516,5 +3516,24 @@
   "twitter": "https://twitter.com/icdragoneyes",
   "youtube": "",
   "submittableId": ""
+  },
+  {
+  "id": "relinkdxyz",
+  "name": "{r}elinkd",
+  "oneLiner": "Digital identity + scores protocol and dapp", 
+  "website": "https://relinkd.xyz/",
+  "tags": [
+    "DID", 
```

Comment:
Can you please check this list for the allowed tags? https://github.com/dfinity/portal?tab=readme-ov-file#tags

I believe `Tools/Infrastructure` is the best tag for {r}elinkd.

Reply:
thanks for the information, updated tags

Reply:
Thank you so much!